clients/src/main/java/org/apache/kafka/clients/consumer/internals/OffsetFetcher.java

```diff
@@ -278,11 +291,13 @@ private void validatePositionsAsync(Map<TopicPartition, FetchPosition> partition
                     offsetsForLeaderEpochClient.sendAsyncRequest(node, fetchPositions);
 
             future.addListener(new RequestFutureListener<OffsetForEpochResult>() {
+                private long attempts = 0L;
                 @Override
                 public void onSuccess(OffsetForEpochResult offsetsResult) {
                     List<SubscriptionState.LogTruncation> truncations = new ArrayList<>();
                     if (!offsetsResult.partitionsToRetry().isEmpty()) {
-                        subscriptions.setNextAllowedRetry(offsetsResult.partitionsToRetry(), time.milliseconds() + retryBackoffMs);
+                        subscriptions.setNextAllowedRetry(offsetsResult.partitionsToRetry(),
```

Comment:
Same question as the above. Does this really do exponential backoff since attempts is 0 for every new request?